docs/relevance.md

```diff
@@ -0,0 +1,21 @@
+# Search results and relevance ranking
+
+Some documentation about existing use cases relevance ranking is helpful when
+new requests come in to make changes to relevance. We
+want to ensure any work we do to address new requirements leaves
+previously-implemented requirements fully functional.
+
+This documentation does not cover issues about records that didn't appear in
+search results because certain fields weren't indexed at all or weren't indexed into the
+correct type of field.
+
+[Closed issues with Search Results label](https://github.com/pulibrary/pulfalight/issues?q=label%3A%22Search+Results%22+is%3Aclosed)
+
+## Current relevance functionality
+
+* #146, #560 Any collection title that contains the search terms should have prioritized ranking, and if the collection title starts with the search terms, it should have highest ranking.
```

Comment:
question: I wonder if these will link out the relevant PRs by default? I tried in preview mode and they didn't.